docs/wallet/web-wallet.md

```diff
@@ -116,3 +116,97 @@ The faucet is only available once in a given time period. Other alternatives for
 
 - request tokens on [Telegram - Validators chat](https://t.me/MultiversXValidators)
 - use a third-party faucet, such as [https://r3d4.fr/faucet](https://r3d4.fr/faucet)
+
+## **Guardian**
+
+:::note
+In case your balance is 0 eGLD, the **Guardian** option is not displayed.
+:::
+
+:::note
+All screenshots have been taken from an internal private wallet on a private internal blockchain. The design of the feature may change for the final release on mainnet.
+:::
+
+Starting with Altair release, a new section for Guardian feature is available in the wallet interface:
+
+![img](/wallet/web-wallet/guardian_feature1.png)
+
+### **Registering the Guardian**
+
+First step should be registering your guardian. After selecting the Guardian feature from the menu, you should be able to set it:
+
+![img](/wallet/web-wallet/guardian_step1.png)
+
+In order to set a Guardian you should use a software-based authenticator. This tool should be able to generate two-factor authentication (2FA's) codes. After installing your authenticator, use it for scanning the QR code and introduce the Guardian Code in the designated fields (see picture below). 
+
+:::important
+If this is the first time when you are doing the registration, you don't have to select the **I cannot access my guardian**.
+:::
```

Comment:
This should be front-end's job to make sure not to display this info. An user should not be prompted with "false/impossible" options